art_web_app/views.py

```diff
@@ -0,0 +1,8 @@
+from django.shortcuts import render
+
+
+def home_view(request):
+    context = {
+        'user':request.user,
+    }
+    return render(request, "home.html", context)
```

Comment:
You don't have to send the user in context. You'll have access to `user` directly through Django's context processors.